docs/data-sources/cluster.md

```diff
@@ -21,45 +94,46 @@ Sample cluster data source in the BigAnimal terraform provider .
 
 ### Read-Only
 
-- `allowed_ip_ranges` (Block List) Allowed IP ranges (see [below for nested schema](#nestedblock--allowed_ip_ranges))
+- `allowed_ip_ranges` (Block List) Allowed IP ranges. (see [below for nested schema](#nestedblock--allowed_ip_ranges))
```

Comment:
```suggestion
- `allowed_ip_ranges` (Block List) Allowed IP ranges. See [nested schema](#nestedblock--allowed_ip_ranges).
```

Suggest you make this same change throughout. If you're comfortable with this change, we can make it for you.

Reply:
So, those changes come from the documentation generator code we're using. The tool is called tfplugindocs, and [here](https://github.com/hashicorp/terraform-plugin-docs/blob/0aa71ca403e91fee06e4e8f0f5353043ef4d5077/schemamd/render.go#L102) are some examples of how they're adding these `(see [below for nested schema]...` changes.  

That's the tool's default. We can still fix it in various ways: 
1. We can update the generated docs, but it will be overwritten every time we generate the docs again. Likewise, we can automate this post-processing. 
2. We can communicate with the upstream and prose a PR for the change. That would probably take some time. 
3. We can create a GitHub Issue and discuss how we can solve it in the future. 

If it's not a dealbreaker, I favor the 3rd option.

Reply:
Option 3 sounds good to me, but I defer to Dee Dee.

Reply:
One minor suggestion. If we leave the generated references as is, perhaps we can move the period to after the generated reference. For example: "Allowed IP ranges (see [below for nested schema](#nestedblock--allowed_ip_ranges))."

Reply:
I do agree with that. I was thinking of creating a GitHub Issue, as I proposed above. But, I don't know how to describe the problem and a possible solution. @dwicinas, may I ask, if you find it useful, can you create a GitHub Issue to this repository and describe the issue? It's also fine for me that we tackle this issue later, when we find it necessary to fix.